packages/components/package.json

```diff
@@ -10,6 +10,7 @@
   "homepage": "https://github.com/chanzuckerberg/edu-design-system/tree/main/packages/components",
   "license": "MIT",
   "main": "dist/index.js",
+  "sideEffects": false,
```

Comment:
Here we're basically promising to Webpack that there are no side effects.

Examples of side effects would be a module changing something outside of itself

```js
window.foo = 666;
```

or providing a CSS file for consumers to import

```js
// In Learning Platform or Along or something...

import '@chanzuckerberg/eds-components/some-styles.css';
```

By telling Webpack that there are no side effects, it'll remove unused  files from something like

```js
import {Heading} from '@chanzuckerberg/eds-components';
```

Reply:
I think it also promises there's no shared "global to the module" values like

```js
let count = 0

function increment1() {
  count += 1;
}

function increment2() {
  count += 2;
}
```

Reply:
My understanding was shared globals **_within_** a module are fine, but shared stuff **_between_** modules is not.

Could be wrong about that. Either way, good call out.